src/fontcache.cpp

```diff
@@ -23,11 +23,20 @@
 #include "safeguards.h"
 
 /** Default heights for the different sizes of fonts. */
-static const int _default_font_height[FS_END]   = {10, 6, 18, 10};
+static const int _default_font_height[FS_END]   = {10, 6, 16, 10};
 static const int _default_font_ascender[FS_END] = { 8, 5, 15,  8};
 
+const int _default_font_sane_max_height_limit = 40;
+const int _default_font_min_max_gap = 14;
+// Because of the math that builds the font size sliders _default_font_min_max_gap needs to be an even number:
```

Comment:
We use `/* Regular comment. */` and `a = b; // End of line comment`.

```suggestion
/* Because of the math that builds the font size sliders _default_font_min_max_gap needs to be an even number. */
```